llpc/translator/lib/SPIRV/SPIRVReader.h

```diff
@@ -272,6 +285,19 @@ class SPIRVToLLVM {
     return t;
   }
 
+  Type *getAccessChainRetType(SPIRVValue *v) {
+    auto loc = m_accessChainRetTypeMap.find(v);
+    if (loc != m_accessChainRetTypeMap.end())
+      return loc->second;
+    return nullptr;
+  }
+
+  void addAccessChainRetType(SPIRVValue *v, Type *t) {
```

Comment:
This is only adding in case the SPIRVValue * was not found inside the map, so I'd suggest to call this `tryAdAccessChainRetType` or something.

Reply:
Good point. Will update